### PR TITLE
Add 'main' in package.json

### PR DESCRIPTION
**Pull request by LeeTaeSoon**
Add 'main' in package.json and
add export default in index.js

### CODE REVIEW

**Comment by LeeTaeSoon** [comment]
https://github.com/JeffGuKang/npm-webpack-boilerplate/issues/36
@JeffGuKang  Is this right?

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "name": "YOUR_PROJECTNAME",
   "version": "0.1.0",
   "description": "test",
+  "main": "index.js",
```
Comment (JeffGuKang) [inline package.json]: `src/index.js`

**Review by JeffGuKang — CHANGES_REQUESTED** [review]
`src/index.js` is right.

**Comment by LeeTaeSoon** [comment]
Changed

**Comment by TylerShin** [comment]
🍶

**Pull request merged**